src/Basic/VGGType.h

```diff
@@ -0,0 +1,72 @@
+#pragma once
+namespace VGG
+{
+
+enum ObjectType
+{
+  VGG_LAYER = 0,
+  VGG_PATH,
+  VGG_IMAGE,
+  VGG_TEXT,
+  VGG_ARTBOARD,
+  VGG_GROUP,
+  VGG_CONTOUR,
+  VGG_MASTER,
+  VGG_INSTANCE
+};
+
+enum BoolOp
+{
+  BO_Union,        // union
+  BO_substraction, // substraction
+  BO_intersection, // intersection
+  BO_exclusion,    // exclusion
```

Comment:
Why not `BO_Substraction` `BO_Intersection` `BO_Exclusion`? These enums are not consistent with others' naming rule.

Reply:
It's typo. My fault.